common/beerocks/bwl/shared/nl80211_client_impl.cpp

```diff
@@ -50,13 +50,11 @@ nl80211_client_impl::nl80211_client_impl(std::unique_ptr<nl80211_socket> socket)
 
 bool nl80211_client_impl::get_radio_info(const std::string &interface_name, radio_info &radio_info)
 {
-    struct channels_ctx {
-        int last_band;
-        bool width_40;
-        bool width_80;
-        bool width_80_80;
-        bool width_160;
-    } ctx{};
+    int last_band    = -1;
+    bool width_40    = false;
+    bool width_80    = false;
+    bool width_80_80 = false;
+    bool width_160   = false;
```

Comment:
Any reason to define these here?
As far as I can see, they are used only when processing the netlink response.

Reply:
This was added to my PR but is part of #1053